ooiservices/app/uframe/plot_tools.py

```diff
@@ -685,16 +700,25 @@ def plot_3d_scatter(self, fig, ax, x, y, z, title='', xlabel='', ylabel='', zlab
             axis_font = axis_font_default
 
         cmap = plt.cm.jet
```

Comment:
They've asked us to use Red-Blue Divergent as the colormap for this.  Should just be plt.cm.RdBu